src/mainboard/pcengines/apu3/mainboard.c

```diff
@@ -179,7 +179,7 @@ static void mainboard_enable(device_t dev)
 		total_mem += (bsp_topmem2() / (1024 * 1024)) - 4 * 1024;
 
 	printk(BIOS_ALERT, CONFIG_MAINBOARD_PART_NUMBER "\n");
-	printk(BIOS_ALERT, "coreboot build %s\n", COREBOOT_BUILD);
+	printk(BIOS_ALERT, "coreboot build %s\n", COREBOOT_DMI_DATE);
```

Comment:
It was to maintain as much similarity to legacy as possible. Those definitions changed in 4.5.x. Take a look at this:
#define COREBOOT_BUILD "Fri Feb 24 14:20:33 UTC 2017
#define COREBOOT_DMI_DATE "02/24/2017"

In legacy version build was `20170224`